install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1271,105 +1271,13 @@ endif::[]
-ifdef::openshift-origin[]
-1.3+.
-endif::[]
-ifdef::openshift-enterprise[]
-3.5+.
-endif::[]
+Manual upgrade steps for logging deployments are no longer available starting in
+{product-title} 3.5.
 ====
 
-. Ensure you are working in the project where the EFK stack was previously
-deployed. For example, if the project is named *logging*:
-+
-----
-$ oc project logging
-----
-
-. Recreate the deployer templates for service accounts and running the deployer:
-+
-ifdef::openshift-enterprise[]
-----
-$ oc apply -n openshift -f \
-    /usr/share/ansible/openshift-ansible/roles/openshift_hosted_templates/files/v1.5/enterprise/logging-deployer.yaml
-----
-endif::[]
-ifdef::openshift-origin[]
-----
-$ oc apply -n openshift -f \
-    https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/deployer/deployer.yaml
-----
-endif::[]
-
-. Generate any missing service accounts and roles:
-+
-----
-$ oc process logging-deployer-account-template | oc apply -f -
-----
-
-. Ensure that the cluster role `oauth-editor` is assigned to the *logging-deployer*
-service account:
-+
-----
-$ oadm policy add-cluster-role-to-user oauth-editor \
-       system:serviceaccount:logging:logging-deployer
-----
-
-. Ensure that the cluster role `rolebinding-reader` is assigned to the
-*aggregated-logging-elasticsearch* service account where `logging` is the namespace
-with aggregated logging installed:
-+
-----
-$ oadm policy add-cluster-role-to-user rolebinding-reader \
-     system:serviceaccount:logging:aggregated-logging-elasticsearch
-----
-
-. If you are upgrading from {product-title}
-ifdef::openshift-origin[]
-1.4 to 1.5,
-endif::[]
-ifdef::openshift-enterprise[]
-3.4 to 3.5,
-endif::[]
-add the *privileged* SCC to the *aggregated-logging-fluentd* service account:
-+
-----
-$ oadm policy add-scc-to-user privileged \
-    system:serviceaccount:logging:aggregated-logging-fluentd
-----
-
-. In preparation for running the deployer, ensure that you have the configurations
-for your current deployment in the xref:../aggregate_logging.adoc#aggregate-logging-specifying-deployer-parameters[*logging-deployer* ConfigMap].
-+
-[IMPORTANT]
-====
-Ensure that your image version is the latest version, not the currently installed
-version.
-====
-
-. Run the deployer with the parameter in `upgrade` mode:
-+
-----
-$ oc new-app logging-deployer-template -p MODE=upgrade
-----
-+
-Running the deployer in this mode handles scaling down the components to
-minimize loss of logs, patching configurations, generating missing secrets and
-keys, and scaling the components back up to their previous replica count.
-+
-[IMPORTANT]
-====
-Due to the privileges needed to label and unlabel a node for controlling the deployment
-of Fluentd pods, the deployer does delete the *logging-fluentd* Daemonset and recreates
-it from the *logging-fluentd-template* template.
-====
+include::install_config/upgrading/automated_upgrades.adoc[tag=automated-logging-upgrade-steps]
```

Comment:
I've removed all of the manual logging upgrade steps, added a "NOTE" box saying as much, and am `include`ing the same steps here from `automated_upgrades.adoc`.